tests/test_checkbox.py

```diff
@@ -0,0 +1,31 @@
+"""
+ Pylibui test suite.
+
+"""
+
+from pylibui.controls import Checkbox
+from tests.utils import WindowTestCase
+
+
+class CheckboxTest(WindowTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.checkbox = Checkbox('my checkbox')
+
+    def test_text_attribute(self):
+        """Tests the checkbox's `text` attribute."""
+        text = 'My new checkbox'
+        self.checkbox.setText(text)
+        self.assertEqual(self.checkbox.getText(), text)
+
+    def test_checked_attribute(self):
+        """Tests the checkbox's `checked` attribute."""
+        self.assertEqual(self.checkbox.getChecked(), False)
+        self.checkbox.setChecked(True)
+        self.assertEqual(self.checkbox.getChecked(), True)
```

Comment:
Ideally, we should only have one assert per test. This means we are testing only one specific thing, and if it fails, we know what really failed.

In this case you should create two tests to validate that we can trigger it to True and then another to validate we can trigger it to false.